MediaBrowser.MediaEncoding/Probing/FFProbeHelpers.cs

```diff
@@ -46,7 +44,7 @@ public static void NormalizeFFProbeResult(InternalMediaInfoResult result)
         /// <param name="tags">The tags.</param>
         /// <param name="key">The key.</param>
         /// <returns>System.String.</returns>
-        public static string GetDictionaryValue(IReadOnlyDictionary<string, string> tags, string key)
+        public static string? GetDictionaryValue(IReadOnlyDictionary<string, string>? tags, string key)
```

Comment:
Is `tags` ever null? And shouldn't this method be replaced with `GetValueOrDefault`?

Reply:
It shouldn't, but Rider warned that it might be called with `null` values, and should thus be checked. They suggested to use a contract or throwing an exception, I just kept the current behavior of handling that value.

Reply:
Re edit: so just remove the check and nullability and trust that it won't break?